dcurve/README.md

```diff
@@ -108,7 +108,7 @@ const grantRoundContributions = fetch(fetchArgs);
 
 // 4. Calculate Distribution
 
-const distribution = clr.calculate(grantRoundContributions);
+const distribution = await clr.calculate(grantRoundContributions);
```

Comment:
Instead of fetching the trust bonus within these calls, does it make sense to make trust bonus part of the input to keep things a little more modular? That way if/when we change the source of the trust bonus, it's less coupled and easier to change. For example something like this:

```typescript
export type Contribution = {
  grantId: string;
  grantAddress: string;
  address: string;
  amount: number;
  trustBonus: number; // add this as part of Contribution objects
};

// step 1
// get trust bonus scores and assign them to your grantRoundContribution objects

// step 2
const distribution = await clr.calculate(grantRoundContributions);
```